ts/utils/hf_utils.py

```diff
@@ -0,0 +1,29 @@
+from huggingface_hub import snapshot_download
+
+
+def download_model(
+    model_id="meta-llama/Meta-Llama-3.1-8B-Instruct",
+    revision="main",
+    model_path=".cache",
+    use_auth_token=True,
+):
+    # Only download pytorch checkpoint files
+    allow_patterns = [
+        "*.json",
+        "*.pt",
+        "*.bin",
+        "*.txt",
+        "*.model",
+        "*.pth",
+        "*.safetensors",
+    ]
+
+    snapshot_path = snapshot_download(
```

Comment:
We should add ignore pattern with original/* as we're downloading the model 2x otherwise.

Reply:
added it. But when I check the actual disk space, I don't see an issue. Not sure if HF is addressing this internally